lambdaclass/curse_of_myrra
# Map all characters sfx and separate sound channels

This PR adds all the sounds for the current characters we have and each ability.

Also refactor the mute/unmute and the slider system of the settings. Separating channels in Music/Sfx.

To test:

- The state of the track (mute/unmute) should persist thought scenes.
- In the Battle Scene you should control both SFX and Music separately.
- Mute the Music with the mute/unmute image then while muted change the volume slider, finally unmute the volumen should be set in the position where you left the slider when it was muted.

closes #895 and #903

- [x] Tested in Unity Desktop
